devfile.yaml

```
@@ -0,0 +1,19 @@
+schemaVersion: 2.1.0
+metadata:
+  name: bash
+components:
+  - name: tools
+    container:
+      image: quay.io/aandriienko/universal:test 
```

I suppose we have to use universal developer image here https://quay.io/repository/devfile/universal-developer-image

---

```suggestion
      image: quay.io/devfile/universal-developer-image:ubi8-latest 
```
@AndrienkoAleksandr your changes in UDI were built and published